src/controllers/activity.controller.js

```diff
@@ -0,0 +1,19 @@
+const Activity = require('../models/Activity.model');
+
+const store = (req, res) => {
```

Comment:
I would call this `save` or `add`?

Store is a bit confusing. Could even call it `saveActivity` or `addActivity`

Whatever you call it, be consistent else where (so for the student table the equivalent should be `saveStudent`, if you use `saveActivity` here

Reply:
I would probably call it `add` seeing as you call your route `addActivity`